src/components/TeamMember.js

```diff
@@ -1,30 +1,27 @@
-import {
-  Box,
-  Card,
-  CardContent,
-  Typography,
-} from "@material-ui/core/";
+import { Box, Card, CardContent, Typography } from "@material-ui/core/";
 import useMyStyles from "../styles/MyStyles";
 import AccountCircleSharpIcon from "@material-ui/icons/AccountCircleSharp";
 
 const TeamMember = (props) => {
   const classes = useMyStyles();
+  const { id, firstName, lastName, url, description } = props.userData;
```

Comment:
You can destruct props in first line of component: ```const TeamMember = ({ id, firstName, lastName, url, description }) => {``` It's better, because you can for example add initial values there

Reply:
I moved destructuring to the first line as suggested.